examples/tower/hello-world/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "hello-world"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib"]
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+shuttle-service = { version = "0.2", features = ["web-tower"] }
+tokio = "0.1"
+serde = { version = "1.0", features = ["derive"] }
+hyper = { version = "0.14", features = ["server"] }
+tower = { version = "0.4", features = ["make", "util"] }
+http = "0.2"
+http-body = "0.4"
+tower-http = { version = "0.3", features = ["trace", "compression-gzip"] }
```

Comment:
Are all of these used? Like `serde` and `http-body`?

Reply:
fixed